cub/cub/device/dispatch/tuning/tuning_scan.cuh

```diff
@@ -230,6 +313,350 @@ struct sm90_tuning<__uint128_t, primitive_op::yes, primitive_accum::no, accum_si
 #endif
 // clang-format on
 
+template <class ValueT,
+          class AccumT,
+          class OffsetT,
+          op_type OpTypeT,
+          primitive_value PrimitiveValue       = is_primitive_value<ValueT>(),
+          primitive_accum PrimitiveAccumulator = is_primitive_accum<AccumT>(),
+          offset_size OffsetSize               = classify_offset_size<OffsetT>(),
+          value_size ValueSize                 = classify_value_size<ValueT>()>
+struct sm100_tuning;
+
+// sum
+template <class ValueT, class AccumT, class OffsetT>
+struct sm100_tuning<ValueT,
+                    AccumT,
+                    OffsetT,
+                    op_type::plus,
+                    primitive_value::yes,
+                    primitive_accum::yes,
+                    offset_size::_4,
+                    value_size::_1>
```

Comment:
Compared with the sm90 tunings, we switch on the `value_size` here, no the `accum_size`, because that's what we actually also iterate in the benchmark. We do not check the size of `AccumT` here, because we already checked previously whether that size corresponds to the size we would have in the benchmark.